bears/html/HTMLLintBear.py

```diff
@@ -1,33 +1,20 @@
-import re
-
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
-from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.settings.Setting import typed_list
 
 
-class HTMLLintBear(LocalBear, Lint):
-    executable = 'html_lint.py'
-    output_regex = re.compile(
-        r'(?P<line>\d+):(?P<column>\d+):\s'
-        r'(?P<severity>Error|Warning|Info):\s(?P<message>.+)'
-    )
-    severity_map = {
-        "Info": RESULT_SEVERITY.INFO,
-        "Warning": RESULT_SEVERITY.NORMAL,
-        "Error": RESULT_SEVERITY.MAJOR
-    }
-
-    def run(self,
-            filename,
-            file,
-            htmllint_ignore: typed_list(str)=[]):
-        '''
-        Checks the code with `html_lint.py` on each file separately.
+@linter(executable='html_lint.py', output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+): '
+                     r'(?P<severity>Error|Warning|Info): (?P<message>.+)')
```

Comment:
afaik, ``\s`` takes tabs too. are you sure there weren't tabs or anything and that this change won't ruin anything?